docs/links/mautic_api_documentation.py

```diff
@@ -2,6 +2,6 @@
 
 link_name = "Reports API documentation" 
 link_text = "Reports API documentation" 
-link_url = "https://developer.mautic.org/#reports" 
+link_url = "https://devdocs.mautic.org/en/5.x/components/reports.html" 
```

Comment:
There is no content in this page on the dev docs at the moment, only the title. I'm thinking either to:

- Add like a note admonition informing that this page is need to be moved and updated, and give a link to the GitHub legacy repository: https://github.com/mautic/developer-documentation/tree/main/source/includes, **or** 

- Link this to the file in the GitHub legacy repository: https://github.com/mautic/developer-documentation/blob/main/source/includes/_api_endpoint_reports.md